packages/dropdowns/src/styled/field/StyledSelect.ts

```diff
@@ -135,7 +135,6 @@ export const StyledSelect = styled(FauxInput).attrs<IStyledSelectProps>(props =>
 }))<IStyledSelectProps>`
   position: relative;
   cursor: ${props => (props.disabled ? 'default' : 'pointer')};
-  appearance: none;
```

Comment:
Since `type="button` is not applied to the Select `div`, we don't have to apply this style.

Reply:
This style can be removed since `type="button` is no longer applied to the Select Dropdown `div`.